services/configuration/src/transactions.rs

```diff
@@ -410,3 +411,35 @@ impl Transaction for VoteAgainst {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{super::tests::{new_tx_config_vote, new_tx_config_vote_against},
```

Comment:
Is this `super::super::` thing is a good idea at all? Maybe it's better to just start the path from crate's root?

Reply:
Something weird happened to me :sweat: